packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -268,15 +268,23 @@ export class EditorToolbarComponent {
 	/**
 	 * Opens the editor settings.
 	 */
-	async openSettings(): Promise< void > {
-		const label = await this.translateFromPage( settingsButtonLabel );
-		const selector = selectors.settingsButton( label );
+	async openSettings( target: EditorToolbarSettingsButton ): Promise< void > {
+		const editorParent = await this.editor.parent();
 
-		if ( await this.targetIsOpen( selector ) ) {
-			return;
+		let locator: Locator;
+		if ( target === 'Settings' ) {
+			const label = await this.translateFromPage( settingsButtonLabel );
+			const selector = selectors.settingsButton( label );
+
+			if ( await this.targetIsOpen( selector ) ) {
+				return;
+			}
+
+			locator = editorParent.locator( selector );
+		} else {
+			locator = editorParent.getByRole( 'button', { name: 'Jetpack', exact: true } );
```

Comment:
This is the new portion of code. Everything else was untouched to avoid scope creep.

Reply:
**Nit (very-non-blocking):** This is totally fine, but I wonder if it might be cleaner to just implement this as a separate method -- having the two-option parameter and then just completely different paths in an if clause kind of makes me think this might be best as two separate methods. Especially because all sorts of stuff can be added by plugins to that toolbar! So it might grow a little smoother too.

Reply:
Interestingly, I arrived at this solution because I felt having one method that could open any one of x different settings panels was easier than implementing a method per panel!

> having the two-option parameter and then just completely different paths in an if clause kind of makes me think this might be best as two separate methods. 

I did it this way because I didn't want to expand the scope of the changes, which is already moderately substantial relative to the spec itself. Lines 275 - 285 could probably be merged into one `getByRole` call and it _should_ just work. I'll make the change and verify.